index.html

```diff
@@ -0,0 +1,1048 @@
+                <article
+                  class="categories-cards__categories-card list list__column"
+                >
+                  <svg class="categories-card__image">
+                    <use
+                      xlink:href="./images/icons/sprite.svg#presentation"
+                    ></use>
+                  </svg>
+                  <h3 class="categories-card__title heading__article-title">
+                    Slides
+                  </h3>
+                  <div class="categories-card__description list">
+                    <p>Pitch-ready</p>
+                    •
+                    <p>Creative</p>
+                  </div>
+                </article>
+              </a>
+            </li>
+          </ul>
+          <ul
+            class="categories__categories-cards list list__column list__elements"
+          >
+            <li>
+              <a href="./catalog.html" class="categories-cards__link">
+                <article
+                  class="categories-cards__categories-card list list__column"
+                >
+                  <svg class="categories-card__image">
+                    <use
+                      xlink:href="./images/icons/sprite.svg#social-media"
+                    ></use>
+                  </svg>
+                  <h3 class="categories-card__title heading__article-title">
+                    App icons
+                  </h3>
+                  <div class="categories-card__description list">
+                    <p>High-quality</p>
+                    •
+                    <p>Elegant</p>
+                  </div>
+                </article>
+              </a>
+            </li>
+            <li>
+              <a href="./catalog.html" class="categories-cards__link">
+                <article
+                  class="categories-cards__categories-card list list__column"
+                >
+                  <svg class="categories-card__image">
+                    <use
+                      xlink:href="./images/icons/sprite.svg#illustration"
+                    ></use>
+                  </svg>
+                  <h3 class="categories-card__title heading__article-title">
+                    Design tools
+                  </h3>
+                  <div class="categories-card__description list">
+                    <p>Easy to use</p>
+                    •
+                    <p>Unique</p>
+                  </div>
+                </article>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="trending container__section">
+        <header class="trending__trending-header list">
+          <div class="list list__column">
+            <h2 class="trending-header__title heading__section">trending</h2>
+            <strong class="trending-controller__title heading__section-title"
+              >Popular Products</strong
+            >
+          </div>
+          <div class="trending-header__trending-controller">
```

Comment:
This DIV or it's direct child DIV is unnecessary, you can remove one of them, please.

Reply:
Fixed 🥇